protocol/v2/ssv/runner/runner_validations.go

```diff
@@ -15,6 +15,10 @@ func (b *BaseRunner) ValidatePreConsensusMsg(runner Runner, signedMsg *spectypes
 		return errors.New("no running duty")
 	}
 
+	// zap.L().Debug("got partial sig message (pre-consensus)", fields.PubKey(b.Share.ValidatorPubKey), fields.Role(b.BeaconRoleType),
```

Comment:
This is just a log I added to debug something, it's useless now, let's remove it :D

Ditto for the log below it (post-consensus)